internal/config/plugin.go

```diff
@@ -42,23 +45,54 @@ func NewPlugins() Plugins {
 
 // Load K9s plugins.
 func (p Plugins) Load() error {
-	return p.LoadPlugins(K9sPlugins)
+	var pluginDirs []string
+	for _, dataDir := range xdg.DataDirs {
+		pluginDirs = append(pluginDirs, filepath.Join(dataDir, K9sPluginDirectory))
+	}
+	return p.LoadPlugins(K9sPluginsFilePath, pluginDirs)
 }
 
-// LoadPlugins loads plugins from a given file.
-func (p Plugins) LoadPlugins(path string) error {
+// LoadPlugins loads plugins from a given file and a set of plugin directories.
+func (p Plugins) LoadPlugins(path string, pluginDirs []string) error {
 	f, err := os.ReadFile(path)
 	if err != nil {
 		return err
 	}
-
 	var pp Plugins
 	if err := yaml.Unmarshal(f, &pp); err != nil {
 		return err
 	}
+
+	for _, pluginDir := range pluginDirs {
+		pluginFiles, err := os.ReadDir(pluginDir)
+		if err != nil {
+			log.Warn().Msgf("Failed reading plugin path %s; %w", pluginDir, err)
```

Comment:
Msgf does not support err wrapping ;(

Reply:
Ok? Do I want error wrapping? 🤔

Reply:
We're just logging here. We could just use %s to that end?